bitnami/tomcat/templates/_pod.tpl

```diff
@@ -70,6 +70,10 @@ containers:
         key: tomcat-password
   - name: TOMCAT_ALLOW_REMOTE_MANAGEMENT
     value: {{ .Values.tomcatAllowRemoteManagement | quote }}
+  {{- if or .Values.catalinaOpts .Values.metrics.jmx.enabled }}    
+  - name: CATALINA_OPTS
+    value: "{{ .Values.catalinaOpts  }} {{ .Values.metrics.jmx.catalinaOpts }}"
```

Comment:
I think this will create weird spaces in the value if one of the parameters are not set

Reply:
This was not addressed nor replied

Reply:
hello, i have just add a function into _helper to do that. You can reviewing if it's fine for you